_config/config.yml

```diff
@@ -9,6 +9,10 @@ SilverStripe\Admin\LeftAndMain:
   extra_requirements_css:
     - 'silverstripe/campaign-admin: client/dist/styles/bundle.css'
 
-SilverStripe\CMS\Model\SiteTree:
-  extensions:
-    - SilverStripe\CampaignAdmin\SiteTreeExtension
+# Allow users only only have access to the campaign-admin, and not all CMS access
+# i.e. CMS_ACCESS_LeftAndMain, to still be able to manage changesets
+SilverStripe\Versioned\ChangeSet:
+  required_permission:
+    - CMS_ACCESS_CampaignAdmin
+  singular_name: 'Campaign'
+  plural_name: 'Campaigns'
```

Comment:
I removed the singular and plural name changes and didn't see anything change. What is this needed for?

Reply:
https://github.com/silverstripe/silverstripe-campaign-admin/pull/346#discussion_r1980653297 links to a closed PR

Reply:
Have removed, turns out there are translations in Versioned which will call Changesets "Campaigns", so we don't need the config